crawler.py

```diff
@@ -229,13 +229,23 @@ def run_all(driver, data_folder):
     data_folder = str(uuid.uuid4())
     run_all(driver, data_folder)
 
+    sys.path.insert(0, 'tools/mozbuild/codecoverage')
+
+    from lcov_rewriter import LcovFileRewriter
+
+    jsvm_output_file = 'jsvm_lcov_output.info'
+    jsvm_files = [os.path.join(jsvm_dir, e) for e in os.listdir(jsvm_dir)]
+    rewriter = LcovFileRewriter(os.path.join('tools', 'chrome-map.json'))
+    rewriter.rewrite_files(jsvm_files, jsvm_output_file, '')
```

Comment:
We should remove the jsvm_files after rewriting them into jsvm_output_file, and remove jsvm_output_file at the end (when we also remove code-coverage-gcda.zip.

Reply:
`jsvm_files` are in the temporary directory created with context manager - aren't they deleted automatically by OS?

Reply:
Yes, I forgot `jsvm_dir` was a temporary directory.